Update frequently-asked-questions.rst

aj07 opened the pull request:
Fixes #4339

aj07 committed:
Update frequently-asked-questions.rst

axic commented:
Can you please use more meaningful commit messages? :)

axic commented on docs/frequently-asked-questions.rst:
```diff
@@ -414,7 +414,7 @@ This is a very interesting question. Suppose that we have a contract field set u
     }
 
-In this case, the mapping of the struct being copied over into the userList is ignored as there is no "list of mapped keys".
+In this case, the mapping of the struct being copied over into the user2 is ignored as there is no "list of mapped keys".
```
`user2` should be enclosed in backticks.

aj07 committed:
Update frequently-asked-questions.rst

In section "What happens to a struct’s mapping when copying over a struct?"  there was typo in doc regarding user2. It's mentioned as UserList as of now which is incorrect.

chriseth commented:
Please remove `the` and squash the commits.

aj07 committed:
Update frequently-asked-questions.rst

In section "What happens to a struct’s mapping when copying over a struct?"  there was typo in doc regarding user2. It's mentioned as UserList as of now which is incorrect.

chriseth commented:
I'm really sorry, but reviewing this over and over again this is taking too many resources.
